contextproject-ms3/src/main/resources/views/playlists.fxml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?import java.net.*?>
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.control.cell.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.text.*?>
+<?import javafx.scene.layout.GridPane?>
+
+<TableView fx:controller="contextproject.controllers.PlaylistsController"
+	xmlns:fx="http://javafx.com/fxml" fx:id="tableView"
+	minWidth = "220"
+	prefWidth = "250">
+	<stylesheets>
+		<URL value="@../stylesheets/library.css" />
+	</stylesheets>
+	<columns>
+		<TableColumn text="Title" fx:id="nameColumn" sortable="false"
+		minWidth ="220"
+		prefWidth="250">
```

Comment:
In terms of code style, don't put spaces in between properties. e.g. this="ok", this = "not ok"